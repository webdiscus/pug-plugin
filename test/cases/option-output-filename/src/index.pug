html
  head
    title Test
    script(src=require('Scripts/main.js'))
  body
    h1 Hello World!
    p= lang