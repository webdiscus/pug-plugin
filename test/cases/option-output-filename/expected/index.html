<!DOCTYPE html><html><head><title>Test</title><script src="js/main.31c897f5.js"></script></head><body><h1>Hello World!</h1><p>en</p></body></html>